src/test/scala/uk/gov/hmrc/perftests/uknwauthcheckerapi/simulations/BaseSimulation.scala

```diff
@@ -21,38 +21,31 @@ import io.gatling.http.Predef._
 import io.gatling.http.request.builder.HttpRequestBuilder
 import io.netty.handler.codec.http.HttpResponseStatus
 import play.api.http.MimeTypes
-import play.api.libs.json.JsValue
+import play.api.libs.json.Json
 import uk.gov.hmrc.performance.conf.ServicesConfiguration
-import uk.gov.hmrc.perftests.uknwauthcheckerapi.requests.PerformanceRequests._
-import uk.gov.hmrc.perftests.uknwauthcheckerapi.util.JsonGetter
+import uk.gov.hmrc.perftests.uknwauthcheckerapi.models.AuthorisationRequest
+import uk.gov.hmrc.perftests.uknwauthcheckerapi.util.generators.EoriGenerator
 
-object BaseSimulation extends ServicesConfiguration with JsonGetter {
+trait BaseSimulation extends ServicesConfiguration with EoriGenerator {
 
   val baseUrl: String = baseUrlFor("uknw-auth-checker-api")
   val route:   String = "/authorisations"
   private val acceptType = "application/vnd.hmrc.1.0+json"
   private val bearerToken: String = s"$${accessToken}"
 
-  private def getHttpRequest(payload: JsValue) =
+  def getHttpRequest(load: Int): HttpRequestBuilder = {
+    val request: AuthorisationRequest = AuthorisationRequest(useEoriGenerator(load, Some(load)))
```

Comment:
Minor change, but this can be changed to as it will default to all authorisedEoris when passing one parameter

val request: AuthorisationRequest = AuthorisationRequest(useEoriGenerator(load))

Reply:
Modified now @Reece-Carruthers